src/System.Net.Http/tests/FunctionalTests/Configurations.props

```diff
@@ -6,6 +6,7 @@
       netcoreapp-Unix;
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
```

Comment:
See my other comment- I'm dependent on that change for the PlatformDetection. This change is benign from a 2.0 perspective.